stress-test/pipeline.yaml

```diff
@@ -0,0 +1,140 @@
+      repository: arbourd/concourse-slack-alert-resource
+      tag: latest
+
+resources:
+
+##############################################################################
+# Git Repositories
+##############################################################################
+
+  - name: eq-survey-runner-benchmark
+    type: git
+    source:
+      uri: https://github.com/ONSdigital/eq-survey-runner-benchmark.git
+      branch: stress-test
+
+  - name: eq-terraform-load-generator
+    type: git
+    source:
+      uri: https://github.com/ONSdigital/eq-terraform-load-generator.git
+      branch: stress-test
+
+##############################################################################
+# Misc Resources
+##############################################################################
+
+  - name: slack-alert
+    type: slack-notification
+    source:
+      url: ((slack.webhook))
+      channel: '#((slack_channel.name))'
+
+jobs:
+
+##############################################################################
+# Deploy Infrastructure
+##############################################################################
+
+  - name: deploy-infrastructure
+    serial_groups: [run-stress-test]
+    plan:
+      - get: eq-terraform-load-generator
+      - task: Build & Deploy Infrastructure
+        file: 'eq-terraform-load-generator/ci/deploy-infrastructure.yaml'
+        params:
+          <<: *infrastructure_vars
+        vars:
+          image_registry: ((docker.build_image_registry))
+    on_failure:
+      put: slack-alert
+      params:
+        message: Load Generator Infrastructure Deploy Failed
+        alert_type: failed
+    on_success:
+      put: slack-alert
+      params:
+        message: Load Generator Infrastructure Deploy Passed
+        alert_type: success
+
+##############################################################################
+# Run Stress Test
+##############################################################################
+
+  - name: run-stress-test
+    serial_groups: [run-stress-test]
+    plan:
+    - get: eq-survey-runner-benchmark
+    - get: eq-terraform-load-generator
+      passed:
+        - deploy-infrastructure
+      trigger: true
+    - task: Run Survey Runner Stress Test
+      file: 'eq-survey-runner-benchmark/ci/run-benchmark.yaml'
+      params:
+        <<: *benchmark_vars
+      on_failure:
+        put: slack-alert
+        params:
+          message: Survey Runner Stress Test Failed
+          alert_type: failed
+      on_success:
+        put: slack-alert
+        params:
+          message: Survey Runner Stress Test Passed
+          alert_type: success
+
+##############################################################################
+# Destroy Infrastructure
+##############################################################################
+
+  - name: destroy-infrastructure
+    serial_groups: [run-stress-test]
+    plan:
+    - get: eq-terraform-load-generator
+      passed:
+        - run-stress-test
```

Comment:
Since this pipeline is triggered manually, I don't think we need this flag. So allow it to be destroyed without run-stress-test passing.